From ken@new.cgi101.com  Tue May  9 00:41:08 2006
From: Ken Brush <ken@new.cgi101.com>
Date: Mon, 8 May 2006 20:24:12 -0500
To: greg@kroah.com
Cc: <dbudik@ist-traffic.com>
Subject: USB: Add Sieraa Wireless 580 evdo card to airprime.c
Message-ID: <20060509012412.GA9877@cgi101.com>
Content-Disposition: inline

This adds the Sierra Wireless card to airprime.c.

I tested this on my laptop.


Signed-off-by: Ken Brush <ken@cgi101.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/airprime.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/usb/serial/airprime.c
+++ gregkh-2.6/drivers/usb/serial/airprime.c
@@ -18,6 +18,7 @@
 static struct usb_device_id id_table [] = {
 	{ USB_DEVICE(0xf3d, 0x0112) },  /* AirPrime CDMA Wireless PC Card */
 	{ USB_DEVICE(0x1410, 0x1110) }, /* Novatel Wireless Merlin CDMA */
+	{ USB_DEVICE(0x1199, 0x0112) }, /* Sierra Wireless Aircard 580 */
 	{ },
 };
 MODULE_DEVICE_TABLE(usb, id_table);
